mkdocs.yml

```diff
@@ -27,4 +27,5 @@ nav:
   - Pastas & similar:
       - Gnocchi with Pumpkin: "main-courses/pastas/gnocchi_with_pumpkin.md"
   - Breakfast:
-      - Porridge: "breakfast/porridge.md"
\ No newline at end of file
+      - Porridge: "breakfast/porridge.md"
+      - Kladdkaka: "Kladdkaka.md" 
```

Comment:
```suggestion
      - Kladdkaka: "breakfast/Kladdkaka.md" 
```

Reply:
Or maybe we should try to move it to Fika. Who knows, it might work this time :)